### PR TITLE
[DOCS] Camera documentation

### PR DESCRIPTION
**Pull request by mhilbrunner**

### CODE REVIEW

File: doc/classes/Camera.xml [hunk 1]
```diff
@@ -184,6 +200,7 @@
 			<argument index="0" name="mode" type="int" enum="Camera.KeepAspect">
 			</argument>
 			<description>
+				Sets the mode for keeping the aspection ratio. See [code]KEEP_*[/code] constants.
```
Comment (Noshyaar) [inline doc/classes/Camera.xml]: aspect

File: doc/classes/Camera.xml [hunk 2]
```diff
@@ -238,14 +256,19 @@
...
+			Try to keep the aspect ratio when scaling the Camera's viewport to the screen. If not possible, preserve the viewport's width by changing the height. Height is [code]sizey[/code] for orthographic projection, [code]fovy[/code] for perspective projection.
 		</constant>
 		<constant name="KEEP_HEIGHT" value="1">
+			Try to keep the aspect ratio when scaling the Camera's viewport to the screen. If not possible, preserve the viewport's height by changing the width. Width is [code]sizex[/code] for orthographic projection, [code]fovx[/code] for perspective projection.
 		</constant>
 		<constant name="DOPPLER_TRACKING_DISABLED" value="0">
+			Disable Doppler effect simulation (default).
 		</constant>
 		<constant name="DOPPLER_TRACKING_IDLE_STEP" value="1">
+			Simulate Doppler effect by tracking positions of objects that are changed in [code]_process[/code]. Changes in the relative velocity of this Camera compared to those objects affect how Audo is perceived (changing the Audio's [code]pitch shift[/code]).
```
Comment (Noshyaar) [inline doc/classes/Camera.xml]: audio

File: doc/classes/Camera.xml [hunk 3]
```diff
@@ -238,14 +256,19 @@
...
 		</constant>
 		<constant name="DOPPLER_TRACKING_FIXED_STEP" value="2">
+			Simulate Doppler effect by tracking positions of objects that are changed in [code]_physics_process[/code]. Changes in the relative velocity of this Camera compared to those objects affect how Audo is perceived (changing the Audio's [code]pitch shift[/code]).
```
Comment (Noshyaar) [inline doc/classes/Camera.xml]: audio

**Comment by mhilbrunner** [comment]
Fixed! Open for suggestions to improve the descriptions.

File: doc/classes/Camera.xml [hunk 4]
```diff
@@ -76,25 +83,28 @@
...
 			<description>
+				Returns the near clip plane in world space units.
 			</description>
 		</method>
 		<method name="is_current" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
-				Return whether the Camera is the current one in the [Viewport], or plans to become current (if outside the scene tree).
+				Returns whether the Camera is the current one in the [Viewport], or plans to become current (if outside the scene tree).
```
Comment (cbscribe) [inline doc/classes/Camera.xml]: "Returns whether" should be changed to "Returns [code]true[/code]" as you did elsewhere.

Comment (mhilbrunner) [reply doc/classes/Camera.xml]: Fixed :)

**Comment by mhilbrunner** [comment]
Fixed!

**Pull request merged**